### YamlMime:ManagedReference
items:
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException
  id: WorkflowTerminatedException
  children:
  - System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor
  - System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor(System.String)
  - System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor(System.String,System.Exception)
  langs:
  - csharp
  name: WorkflowTerminatedException
  nameWithType: WorkflowTerminatedException
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException
  type: Class
  summary: "Rappresenta il <xref:System.Exception>che viene generato quando un flusso di lavoro viene terminato.</xref:System.Exception> Questa classe non può essere ereditata."
  remarks: >-
    > [!IMPORTANT]

    >  [!INCLUDE[DeprecatedContent](~/add/includes/ajax-current-ext-md.md)]
  syntax:
    content: >-
      [System.Obsolete("The System.Workflow.* types are deprecated.  Instead, please use the new types from System.Activities.*")]

      public sealed class WorkflowTerminatedException : Exception
  inheritance:
  - System.Exception
  implements: []
  inheritedMembers: []
  platform:
  - net462
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor
  id: '#ctor'
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  langs:
  - csharp
  name: WorkflowTerminatedException()
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException()
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException.WorkflowTerminatedException()
  type: Constructor
  assemblies:
  - System.Workflow.ComponentModel
  namespace: System.Workflow.ComponentModel
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.Workflow.ComponentModel.WorkflowTerminatedException&quot;> </xref> classe."
  syntax:
    content: public WorkflowTerminatedException ();
    parameters: []
  overload: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor(System.String)
  id: '#ctor(System.String)'
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  langs:
  - csharp
  name: WorkflowTerminatedException(String)
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException(String)
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException.WorkflowTerminatedException(String)
  type: Constructor
  assemblies:
  - System.Workflow.ComponentModel
  namespace: System.Workflow.ComponentModel
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.Workflow.ComponentModel.WorkflowTerminatedException&quot;> </xref> (classe), con un messaggio di errore specificato."
  syntax:
    content: public WorkflowTerminatedException (string message);
    parameters:
    - id: message
      type: System.String
      description: "Il messaggio che descrive l&quot;errore."
  overload: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor*
  exceptions: []
  platform:
  - net462
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor(System.String,System.Exception)
  id: '#ctor(System.String,System.Exception)'
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  langs:
  - csharp
  name: WorkflowTerminatedException(String,Exception)
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException(String,Exception)
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException.WorkflowTerminatedException(String,Exception)
  type: Constructor
  assemblies:
  - System.Workflow.ComponentModel
  namespace: System.Workflow.ComponentModel
  summary: "Inizializza una nuova istanza di <xref href=&quot;System.Workflow.ComponentModel.WorkflowTerminatedException&quot;> </xref> (classe), con un messaggio di errore specificato e un riferimento a interna <xref:System.Exception>che è la causa di questo <xref:System.Exception>.</xref:System.Exception> </xref:System.Exception>"
  syntax:
    content: public WorkflowTerminatedException (string message, Exception exception);
    parameters:
    - id: message
      type: System.String
      description: "Il messaggio che descrive l&quot;errore."
    - id: exception
      type: System.Exception
      description: "Interna <xref:System.Exception>riferimento.</xref:System.Exception>"
  overload: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor*
  exceptions: []
  platform:
  - net462
references:
- uid: System.Exception
  isExternal: false
  name: System.Exception
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  isExternal: false
  name: WorkflowTerminatedException()
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException()
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException.WorkflowTerminatedException()
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor(System.String)
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  isExternal: false
  name: WorkflowTerminatedException(String)
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException(String)
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException.WorkflowTerminatedException(String)
- uid: System.String
  parent: System
  isExternal: true
  name: String
  nameWithType: String
  fullName: System.String
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor(System.String,System.Exception)
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  isExternal: false
  name: WorkflowTerminatedException(String,Exception)
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException(String,Exception)
  fullName: System.Workflow.ComponentModel.WorkflowTerminatedException.WorkflowTerminatedException(String,Exception)
- uid: System.Workflow.ComponentModel.WorkflowTerminatedException.#ctor*
  parent: System.Workflow.ComponentModel.WorkflowTerminatedException
  isExternal: false
  name: WorkflowTerminatedException
  nameWithType: WorkflowTerminatedException.WorkflowTerminatedException
